src/geo/ui/widgets/formula/animation_template.tpl

```diff
@@ -0,0 +1 @@
+<%= prefix %><%= value %><%= suffix %>
```

Comment:
Ops, sorry, I had an issue with the separation between prefix, value and suffix, tried several things and forgot to change the template back to the safe setting. It's fixed now.